docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+
+All the service parameters (model, schedule, input-output) are defined in a config file.
+
+Single config file supports only one model, but it's totally OK to run multiple vmanomaly processes in parallel, each using its own config.
+
+VmAnomaly periodically queries user-specified metrics, computes an **anomaly score** for them, based on how well they fit a predicted distribution, taking into account periodical data patterns with trends and/or seasonality, and pushes back the computed "anomaly score" to VictoriaMetrics. Then, users can enable alerting rules based on the "anomaly score" with VmAlert.
+
+
+## What is VmAlert?
+VMalert is an alerting tool for VictoriaMetrics. It executes a list of the given alerting or recording rules against configured `-datasource.url` compatible with Prometheus HTTP API.
+
+Alerting rules allow you to define alert conditions based on Prometheus expression language expressions and to send notifications about firing alerts to an external service. Whenever the alert expression results in one or more vector elements at a given point in time, the alert counts as active for these elements' label sets.
+
+## How does VmAnomaly works with VmAlert?
+Compared to classical alerting rules, anomaly detection is more "hands-off" i.e. it allows users to avoid setting up manual alerting rules set up and catching anomalies that were not expected to happen. In other words, by setting up alerting rules, a user must know what to look for, ahead of time, while anomaly detection looks for any deviations from past behavior. In addition to that, setting up alerting rules manually has been proven to be tedious and error-prone, while anomaly detection can be easier to set up, and use the same model for different metrics.
+
+Practical use case is to put anomaly score generated by VmAnomaly into alerting rules with some threshold. Anomaly score is designed to fall between 0 and 1 if model consider that datapoint is following usual pattern, and if it exceeded 1 the datapoint is abnormal.
+
+
+
+
+
+### In this tutorial we are going to:
+  - Configure docker-compose file with all needed services (VictoriaMetrics, VmAlert, VmAgent, Grafana, NodeExporter and VmAnomaly).
+  - Explore configuration files for VmAnomaly and VmAlert.
+  - Run our own VictoriaMetrics database with data scraped from NodeExporter.
+  - Explore data for analysis in Grafana.
+  - Explore VmAnomaly results.
+  - Explore VmAlert alerts
+
+ 
+
+
+_____________________________
+
+## Data to analyze
+Let's talk about data used for anomaly detection in this tutorial. 
+We are going to collect our own CPU usage data with node-exporter into the VictoriaMertics database.
+
+On a Node exporters' metrics page, part of the output looks like this:
+```
+# HELP node_cpu_seconds_total Seconds the CPUs spent in each mode.
```

Comment:
I think we won't lose anything if we make a shorter example. 5-7 time series is pretty enough to show the nature of data